frontend/src/scenes/dashboard/DashboardItem.tsx

```diff
@@ -285,7 +287,7 @@ export function DashboardItem({
             } ph-no-capture`}
             {...longPressProps}
             data-attr={'dashboard-item-' + index}
-            style={{ border: isHighlighted ? '2px solid var(--primary)' : undefined }}
+            style={{ border: isHighlighted ? '2px solid var(--primary)' : undefined, opacity: isReloading ? 0.5 : 1 }}
```

Comment:
This `opacity: isReloading ? 0.5 : 1` is a quick hack. Willing to change to something better, but thought it's fine to start given the other inline styles.